lib/node_modules/@stdlib/stats/base/varianceyc/test/test.ndarray.js

```diff
@@ -119,8 +192,26 @@ tape( 'if provided a `correction` parameter yielding `N-correction` less than or
 	t.end();
 });
 
+tape( 'if provided a `correction` parameter yielding `N-correction` less than or equal to `0`, the function returns `NaN (accessor)`', function test( t ) {
+	var x;
+	var v;
+
+	x = toAccessorArray([ 1.0, -2.0, -4.0, 5.0, 3.0 ]);
+
+	v = varianceyc( x.length, x.length, x, 1, 0 );
+	t.strictEqual( isnan( v ), true, 'returns expected value when n = 0' );
```

Comment:
```suggestion
	t.strictEqual( isnan( v ), true, 'returns expected value' );
```

No bespoke messages unless actually necessary. Applies throughout this PR.